string  chunk_name      # Name of the chunk node
bool    value           # true to enable, false to disable
---
